src/scss/attendee/layer.scss

```
@@ -34,6 +34,8 @@
 // Major application or page wide z-index values should follow here
 // Minor ones constrained to their own component can be provided in separate files
 $-z-index--baseline: 1000;
-$-z-index--header: 50;
-$z-index--attendee: above($-z-index--baseline);
+$-z-index--header: above($-z-index--baseline + 50);
+$-z-index--chatbox: above($-z-index--baseline + 100);
+$z-index--attendee: above($-z-index--baseline + 200);
 $z-index--header: level($-z-index--header);
+$z-index--chatbox: level($-z-index--baseline + 70);
```

Please don't add hard-coded values, this can easily (and more robustly) be replaced with 
```suggestion
$-z-index--header: above($-z-index--baseline);
$-z-index--chatbox: above($-z-index--header);
$-z-index--attendee: above($-z-index--chatbox);
$z-index--header: level($-z-index--header);
$z-index--chatbox: level($-z-index--chatbox);
```

But also, there shouldn't be a private and a public version of these variables. If they're needed for multiple different components, sure, this is a good place to have `$z-index--` ones, otherwise `$-z-index--` ones can be moved in the one file they are required.

---

[ce03ad0](https://github.com/sparkletown/sparkle/pull/2973/commits/ce03ad0c063b92163c5039c84f70f34635278d0b)